--- slicot-5.0+20101122/debian-orig/rules	2018-10-27 09:56:33.000000000 +0200
+++ slicot-5.0+20101122/debian/rules	2020-08-12 18:51:23.645199247 +0200
@@ -13,7 +13,7 @@
 endif
 
 %:
-	dh $@ 
+	dh $@ --without autoreconf
 
 F77=gfortran
 LDFLAGS += -Wl,--as-needed
@@ -24,15 +24,10 @@
 SLICOT_SRC=$(sort $(shell echo src/*.f))
 SLICOT_OBJ=$(SLICOT_SRC:.f=.o)
 
-debian/shared_dir debian/static_dir debian/shared64_dir:
+debian/shared_dir debian/shared64_dir:
 	mkdir -p $(subst _dir,,$(@))
 	touch $@
 
-$(subst src/,debian/static/,$(SLICOT_OBJ)): \
-  debian/static/%.o : src/%.f debian/static_dir
-
-	$(F77) $(FFLAGS) -c $< -o $@
-
 $(subst src/,debian/shared/,$(SLICOT_OBJ)): \
   debian/shared/%.o : src/%.f debian/shared_dir
 
@@ -52,16 +47,7 @@
 libslicot.so.$(VERS): $(subst src/,debian/shared/,$(SLICOT_OBJ))
 	$(F77) $(LDFLAGS) -shared -Wl,-soname=libslicot.so.$(SO) -o $@ $^ -lblas -llapack
 
-libslicot.a: $(subst src/,debian/static/,$(SLICOT_OBJ))
-	ar r $@ $^
-
-libslicot_pic.a: $(subst src/,debian/shared/,$(SLICOT_OBJ))
-	ar r $@ $^
-
-libslicot64_pic.a: $(subst src/,debian/shared64/,$(SLICOT_OBJ))
-	ar r $@ $^
-
-override_dh_auto_build-arch: libslicot.so libslicot.a libslicot_pic.a libslicot64_pic.a
+override_dh_auto_build-arch: libslicot.so
 
 override_dh_auto_build-indep:
 
