--- trunk/wscript.orig	2020-08-13 18:39:35.374454971 +0200
+++ trunk/wscript	2020-08-13 18:39:56.985455903 +0200
@@ -375,7 +375,6 @@
     conf.env['DESKAPPS_DIR'] = os.path.normpath(os.path.join(conf.env['SHAREDIR'], 'applications'))
     conf.env['BIN_NAME'] = APPNAME
     if opt.lv2:
-        conf.gxload('strip')
         conf.gxload('lv2')
     conf.env['LADSPA'] = opt.ladspa
     conf.env['NEW_LADSPA'] = opt.new_ladspa
