diff -u -r xoscope-code.orig/alsa.c xoscope-code/alsa.c
--- xoscope-code.orig/alsa.c	2018-06-21 03:55:00.084921758 +0200
+++ xoscope-code/alsa.c	2018-06-21 04:37:14.066031049 +0200
@@ -304,19 +304,31 @@
     int newrate = sound_card_rate;
 
     if (dir > 0) {
-        if (sound_card_rate > 16500)
+        if (sound_card_rate > 92000)
+            newrate = 192000;
+        else if (sound_card_rate > 68000)
+            newrate = 96000;
+        else if (sound_card_rate > 46000)
+            newrate = 88200;
+        else if (sound_card_rate > 38000)
+            newrate = 48000;
+        else if (sound_card_rate > 27000)
             newrate = 44100;
-        else if (sound_card_rate > 9500)
-            newrate = 22050;
         else
-            newrate = 11025;
+            newrate = 32000;
     } else {
-        if (sound_card_rate < 16500)
-            newrate = 8000;
-        else if (sound_card_rate < 33000)
-            newrate = 11025;
-        else
+        if (sound_card_rate < 38000)
             newrate = 22050;
+        else if (sound_card_rate < 46000)
+            newrate = 32000;
+        else if (sound_card_rate < 68000)
+            newrate = 44100;
+        else if (sound_card_rate < 92000)
+            newrate = 48000;
+        else if (sound_card_rate < 140000)
+            newrate = 88200;
+        else
+            newrate = 96000;
     }
 
     if (newrate != sound_card_rate) {
diff -u -r xoscope-code.orig/configure.ac xoscope-code/configure.ac
--- xoscope-code.orig/configure.ac	2018-06-21 03:55:00.089921758 +0200
+++ xoscope-code/configure.ac	2018-06-21 04:47:51.107058525 +0200
@@ -112,7 +112,7 @@
 dnl Program defaults for the command-line options (original values)
 
 AC_DEFINE(DEF_A, 1, [active channel])
-AC_DEFINE(DEF_R, 44100, [sample rate in Hz])
+AC_DEFINE(DEF_R, 48000, [sample rate in Hz])
 AC_DEFINE(DEF_S, 10, [time scale in ms/div])
 AC_DEFINE(DEF_T, "0:0:x", [trigger level])
 AC_DEFINE(DEF_L, "1:1:0", [cursor lines])
diff -u -r xoscope-code.orig/xoscope.1 xoscope-code/xoscope.1
--- xoscope-code.orig/xoscope.1	2018-06-21 03:55:00.187921762 +0200
+++ xoscope-code/xoscope.1	2018-06-21 04:41:10.202041234 +0200
@@ -22,7 +22,7 @@
 .TP 0.5i
 .B Soundcard
 Audio sound recording via Advanced Linux Sound Architecture (ALSA).
-Two 8-bit analog channels at 8000 S/s to 44100 S/s.  Left and right
+Two 8-bit analog channels at 22050 S/s to 192000 S/s.  Left and right
 audio is connected to A and B inputs respectively.  Use an external
 mixer program to select which sound inputs to record.  AC coupled,
 voltages unknown, 256K sample memory.
@@ -276,7 +276,7 @@
 .TP 0.5i
 .B -r <rate>
 Sampling Rate in samples per second.  For the sound card, current valid values
-are 8000, 11025, 22050, or 44100.
+are 22050, 32000, 44100, 44100, 48000, 88200, 96000 or 192000.
 
 .TP 0.5i
 .B -s <scale>
@@ -383,8 +383,8 @@
 
 .SH LIMITATIONS
 
-The sound card should be capable of 44100 Hz sampling via the sound
-drivers.  You must use an external mixer program to select the input
+The sound card should be capable of 22050 to 192000 Hz sampling via the
+sound drivers. You must use an external mixer program to select the input
 source device, level, etc.  Since these unknowns affect the amplitude,
 there is no reference to voltage on the Y axis; it is in fact,
 unknown.  Instead you're given the scale in pixels per sample unit.
diff -u -r xoscope-code.orig/xoscope_gtk.c xoscope-code/xoscope_gtk.c
--- xoscope-code.orig/xoscope_gtk.c	2018-06-21 03:55:00.282921766 +0200
+++ xoscope-code/xoscope_gtk.c	2018-06-21 04:46:39.366055431 +0200
@@ -458,9 +458,9 @@
     glist = g_list_append(glist, "$ch1 - $ch1[0]");
     glist = g_list_append(glist, "$ch1 / ($ch2 || 1)");
     glist = g_list_append(glist, "$ch1 > $ch2 ? $ch1 : $ch2");
-    glist = g_list_append(glist, "$t / (44100/2/250) % 2 ? 64 : -64");
-    glist = g_list_append(glist, "sin($t * $pi / (44100/2/250)) * 64");
-    glist = g_list_append(glist, "cos($t * $pi / (44100/2/250)) * 64");
+    glist = g_list_append(glist, "$t / (48000/2/250) % 2 ? 64 : -64");
+    glist = g_list_append(glist, "sin($t * $pi / (48000/2/250)) * 64");
+    glist = g_list_append(glist, "cos($t * $pi / (48000/2/250)) * 64");
 
     window = gtk_dialog_new();
     run = gtk_button_new_with_label("  Run  ");
